config/default.yaml

```diff
@@ -232,5 +232,7 @@ ecosystem:
     badges: https://img.shields.io/badge/build-{{status}}-{{color}}.svg
     # Default registry to pull build containers from. Uses Docker Hub if nothing/empty string is provided
     dockerRegistry: ""
+    # Pushgateway URL for Prometheus
+    pushgatewayUrl: http://pushgateway.example.org:9091
```

Comment:
Can we comment this out? If we don't set the ENV, the executor will try to use this.